ListableUI/Sources/Behavior.swift

```diff
@@ -80,6 +80,11 @@ extension Behavior
         
         /// The `contentInset` of the list is adjusted when the keyboard appears or disappears.
         case adjustsWhenVisible
+
+        /// Consumer calculates the edge insets and handles setting them
+        /// via the `ListView#customScrollViewInsets` callback. Only use this option
```

Comment:
Probably best to use the DocC standard annotation here.

```suggestion
        /// via the ``ListView/customScrollViewInsets`` callback. Only use this option
```

Fine to leave as is as well of course.

Reply:
good to know!

- [ ] todo